XMLTools.cpp

```diff
@@ -2634,7 +2635,7 @@ void commentSelection() {
   int errflag = validateSelectionForComment(str, sellength);
   if (errflag) {
     wchar_t msg[512];
-    swprintf(msg, 512, L"The current selection covers part only one portion of another comment.\nUncomment process may be not applicable.\n\nDo you want to continue ?", errflag);
+    swprintf(msg, 512, L"The current selection covers part only one portion of another comment.\nUncomment process may be not applicable.\n\nDo you want to continue ? %d", errflag);

```

Comment:
How should the errflag be handled? Just added %d to avoid compiler warning, but maybe the value of the flag should be somehow included to one of the sentences before.